app/Http/Controllers/JwtClaimsSupplier.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Resources\JwtClaims;
+use App\Models\User;
+
+class JwtClaimsSupplier extends Controller
```

Comment:
Why such name?

Reply:
If the name caused confusion, then probably already indicates that it was a bad name. I changed it to the generic `JwtClaimsController`.